cw-orch-daemon/src/senders/cosmos.rs

```diff
@@ -132,6 +132,28 @@ impl Wallet {
         self.options.clone()
     }
 
+    /// Replaces the private key that the [CosmosSender] is using with the provided mnemonic string.
```

Comment:
```suggestion
    /// Replaces the private key that the [CosmosSender] is using with key derived from the provided 24-word mnemonic.
```

Reply:
And as a side-note: have we tried figuring out how to support 12-word mnemonics?